jstests/core/explain_use_backup_plan.js

```diff
@@ -0,0 +1,30 @@
+// Test that the explain will use backup plan if the original winning plan ran out of memory in the
+// "executionStats" mode
+// This test was designed to reproduce SERVER-32721"
+
+load("jstests/libs/analyze_plan.js");
+
+"use strict";
+
+db.foo.drop();
+let bulk = db.foo.initializeUnorderedBulkOp();
+
+for (let i = 0; i < 100000; ++i) {
+    bulk.insert({_id: i, x: i, y: i});
+}
+
+bulk.execute();
+db.foo.ensureIndex({x: 1});
+
+// Configure log level and lower the sort bytes limit.
+db.setLogLevel(5, "query");
```

Comment:
You don't need to set the log level for the test, that was just for your personal info. Please remove this line so the test doesn't generate lots of logs. This will also change the setting for subsequent tests, so it would get quite noisy.